src/Symfony/Component/Validator/ConstraintViolationListInterface.php

```diff
@@ -61,4 +61,11 @@ public function set(int $offset, ConstraintViolationInterface $violation);
      * @param int $offset The offset to remove
      */
     public function remove(int $offset);
+
+    /**
+     * Returns whether the violation list is empty.
+     *
+     * @return bool
+     */
+    public function isEmpty();
```

Comment:
Let's use actual return type declarations whenever possible.

```suggestion
     */
    public function isEmpty(): bool;
```

Reply:
Done, it now uses return type declarations.

Reply:
Fabbot complains already. 🙂 

We remove redundant `@return` annotations.

Reply:
Thanks, I just removed them now. 👍🏻